navigation/libandroid-navigation-ui/src/main/res/values/colors.xml

```diff
@@ -4,5 +4,24 @@
     <color name="mapbox_navigation_route_layer_congestion_yellow">#FFAB65</color>
     <color name="mapbox_navigation_route_layer_congestion_red">#E85552</color>
     <color name="mapbox_navigation_route_shield_layer_color">#2C5F99</color>
+
+    <color name="colorPrimary">@color/mapboxNavy</color>
+    <color name="colorPrimaryDark">@color/mapboxVoyageBlue</color>
+    <color name="colorAccent">@color/mapboxBlue</color>
+
+    <color name="mapboxWhite">#ffffff</color>
```

Comment:
Color values should use a unique name and either match the snack naming convension used for congestion colors or camel case for all values.

Reply:
Removed